common/protob/messages-thp.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto2";
+package hw.trezor.messages.thp;
+
+// Sugar for easier handling in Java
+option java_package = "com.satoshilabs.trezor.lib.protobuf";
+option java_outer_classname = "TrezorMessageThp";
+
```

Comment:
@mmilata Here should possibly be 

`option (include_in_bitcoin_only) = true;`

What do you think?

Reply:
yep definitely